plugin/kubernetes/pod.go

```diff
@@ -65,19 +62,19 @@ func (pdInfo podInfoResult) toMeta() plugin.EntryMetadata {
 	return meta
 }
 
-func (p *pod) fetchLogContent(ctx context.Context) ([]byte, error) {
+func (p *pod) fetchLogContent(ctx context.Context) []byte {
 	req := p.client.CoreV1().Pods(p.ns).GetLogs(p.Name(), &corev1.PodLogOptions{})
 	rdr, err := req.Stream()
 	if err != nil {
-		return nil, err
+		return []byte(fmt.Sprintf("unable to access logs: %v", err))
```

Comment:
Interesting. This is a good way to handle any `List` errors for these types of files.